pkg/ccl/changefeedccl/changefeedbase/settings.go

```diff
@@ -348,11 +348,11 @@ var DefaultLaggingRangesThreshold = 3 * time.Minute
 // DefaultLaggingRangesPollingInterval is the default polling rate at which
 // lagging ranges are checked and metrics are updated.
 var DefaultLaggingRangesPollingInterval = 1 * time.Minute
-
 var Quantize = settings.RegisterDurationSettingWithExplicitUnit(
 	settings.ApplicationLevel,
 	"changefeed.resolved_timestamp.granularity",
 	"the granularity at which changefeed progress are quantized to make tracking more efficient",
-	0,
+	time.Duration(metamorphic.ConstantWithTestRange("changefeed."+
+		"resolved_timestamp.granularity", 0, 0, 20))*time.Second,
```

Comment:
nit: don't break up the setting name
```suggestion
	time.Duration(metamorphic.ConstantWithTestRange(
		"changefeed.resolved_timestamp.granularity", 0, 0, 20))*time.Second,
```

Reply:
Ugh, editors auto-correcting things. I think it's fixed now.